Updated Docker Subscription (#13496)
docker/docker.github.io: subscription/upgrade.md

- /docker-hub/billing/upgrade/
---

This page contains information on how to purchase a Docker paid subscription. It also contains information on how to upgrade your existing subscription and update your billing information.

You can upgrade to a Docker Pro, Team, or a Business subscription from a Personal subscription. When you upgrade to a paid subscription, you can immediately use all the features and entitlements offered in your new subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.

## Purchase a Docker subscription

The following section contains information on how to purchase a Docker  subscription for new customers.

> To purchase Docker subscription:

Reviewer: ```suggestion
To purchase a Docker subscription:
```

1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.

2. Open the drop-down menu next to your username in the top-right corner and select **Billing** to go to the **Billing Details** page. This page displays billing information about your existing user account.

   > **Note**
   >

>    > Ensure you have selected your personal account on the left side of the

Reviewer: ```suggestion
   > Ensure you have selected the correct account on the left side of the
```

     > **Note**
     >
     > You can also specify the number of users you’d like to add to your organization. On the **Organization size** page, the cost for the annual subscription is set by default. If you prefer to change this, select **Pay Monthly**.

5. You’ll also receive an email confirming your new subscription.

6. Open the drop-down menu next to your username in the top-right corner and select **Billing**.

    You can view the information about your new subscription and the total cost
  on the **Billing** page.

## Upgrade to Docker Business

To upgrade to a Docker Business subscription, [Contact Sales](https://www.docker.com/pricing/contact-sales){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.

## Upgrade from a monthly subscription to an annual subscription

To upgrade from a monthly subscription to an annual subscription:

1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account as an organization owner or with your personal account.
2. Open the drop-down menu next to your username in the top-right corner and select **Billing**.

> 3. Select your personal account or organization from the left-hand menu and select the **Payment Methods** tab

Reviewer: ```suggestion
3. Select your personal account or organization from the left-hand menu and select the **Payment Methods** tab.
```
